Core/GDCore/IDE/Events/EventsVariablesFinder.h

```diff
@@ -113,13 +113,36 @@ class EventsVariablesFinder {
    * \return A std::set filled with the values used for all parameters of the
    * specified type
    */
-  static std::set<gd::String> FindArgumentsInEvents(
+  static std::set<gd::String> FindArgumentsInEventsAndDependencies(
       const gd::Platform& platform,
       const gd::Project& project,
       const gd::Layout& layout,
       const gd::EventsList& events,
       const gd::String& parameterType,
       const gd::String& objectName = "");
+
+  /**
+   * Construct a list of the value of the arguments for parameters of type @
+   * parameterType. It doesn't search in events dependencies.
+   *
+   * \param project The project used
+   * \param project The layout used

```

Comment:
nit: I know it was already not properly documented, but this should be:

```suggestion
   *
   * \param platform The platform of the project
   * \param project The project
   * \param layout The layout

```